indexer/bin/indexer.js

```diff
@@ -157,16 +164,24 @@ export default {
       console.log(
         `New proof set rail (proof_set_id=${payload.proof_set_id}, rail_id=${payload.rail_id}, payer=${payload.payer}, payee=${payload.payee}, with_cdn=${payload.with_cdn})`,
       )
+      let isPayerSanctioned
+      if (payload.with_cdn) {
+        isPayerSanctioned = await isAddressSanctioned(
+          CHAINALYSIS_API_KEY,
+          payload.payer,
+        )
+      }
```

Comment:
this can create slightly confusing data in the db, as we can store rows with `with_cdn=false` and `is_payer_sanctioned=false` even though the payer is sanctioned. For now, for correctness, can we always look up whether they are sanctioned?

Reply:
In case `with_cdn = false` we won't check if address is sanctioned and it should default to `null`. I can see how the data can be confusing in that case but I've done this only to save some API calls towards Chainalysis.

Reply:
good point, I didn't see that it would be `null` and not `false`. Wfm!